ori/testing/volume/client.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 OnMetal authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package volume
+
+import (
+	"context"
+
+	ori "github.com/onmetal/onmetal-api/ori/apis/volume/v1alpha1"
+	"google.golang.org/grpc"
+)
+
+func NewFakeRuntimeClient() *FakeRuntimeClient {
```

Comment:
We should follow the same approach we have in `ori-machine`: Define a `RuntimeService` interface that is decoupled from any `grpc` internals, have a `FakeRuntimeService` that implements the `RuntimeService` interface and have a `remoteRuntime` that implements the `RuntimeService` using a GRPC client.

For the implementation in `ori-machine` see:
- [`remoteRuntime` runtime.go](https://github.com/onmetal/onmetal-api/blob/e70f23298bd5a7bcc67206886f3d6df85c73b5e8/ori/remote/machine/runtime.go#L27)
- [`RuntimeService` machine.go](https://github.com/onmetal/onmetal-api/blob/e70f23298bd5a7bcc67206886f3d6df85c73b5e8/ori/apis/machine/machine.go#L23)